classes/patreon_wordpress.php

```diff
@@ -773,9 +773,17 @@ public static function getUserPatronageLevel( $pledge ) {
 		
 		// Check whether the pledge value that comes in is higher than the max entitled tier value - for custom pledge amounts
 		// This is not currency aware, but should cover cases in which there is large difference in between the custom pledge and the local tier level for which the content is locked for
+		if ( isset( $pledge['attributes']['amount_cents'] )) {
+			$pledgeAmount = $pledge['attributes']['amount_cents'];
```

Comment:
The code tests out ok. Can you change the variable name to be consistent with rest of the code and WP org style guide? Ie $pledgeAmount instead of $pledge_amount.